src/helm/proxy/models.py

```diff
@@ -374,6 +374,31 @@ def engine(self) -> str:
         "JavaScript subset of The Stack (v1.1).",
         tags=[CODE_MODEL_TAG],
     ),
+    # Local HuggingFace models
+    Model(
+        group="local",
+        creator_organization="Meta",
+        name="local/llama-7b",
+        display_name="LLaMA (7B)",
+        description="", # TODO add description
+        tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG]
+    ),
+    Model(
+        group="local",
+        creator_organization="CRFM",
+        name="local/alpaca-7b",
+        display_name="Alpaca (7B)",
+        description="", # TODO add description
+        tags=[TEXT_MODEL_TAG, FULL_FUNCTIONALITY_TEXT_MODEL_TAG]
+    ),
+    Model(
+        group="local",
+        creator_organization="OpenAI",
+        name="local/gpt2",
```

Comment:
Is this just for testing? We already having `huggingface/gpt2`. Can we remove this? Presumably one could use the flag for testing?

Reply:
Yep, this was just to test that using a local model without the flag would also work:) removed